recipes-core/systemd/systemd_%.bbappend

```diff
@@ -20,8 +20,6 @@ FILES_${PN} += " \
     ${libdir}/sysctl.d/99-printk.conf \
 "
 
-PACKAGECONFIG_append = " kmod "
```

Comment:
Do you know why this was added here in d1efafb7234affbd221730310dbaf2f2cfbca272 ? What has changed since then?

Reply:
I thought about this and I have no answer. I can only confirm that module is in fact there by default and functions as intended.

Reply:
Tested on ARP?